bin/terminus

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env php
 <?php
 
+ini_set("error_reporting", "E_ALL & ~E_NOTICE & ~E_DEPRECATED & ~E_WARNING");
```

Comment:
I guess this is okay for a phar, but it does make it hard to override the error reporting setting should you ever want to do that. @TeslaDethray do you have an opinion?

Reply:
Not a strong one.

Reply:
I just noticed a typo: `"E_ALL & ~E_NOTICE & ~E_DEPRECATED & ~E_WARNING"` is quoted. You want unquoted here.

Maybe we should read an environment variable, TERMINUS_ERROR_REPORTING, and use the default value iff the env var is unset? Not super convenient, as the env var would have to be an integer, but better than not having it.